libs/evm-protocols/src/common-protocol/chainConfig.ts

```diff
@@ -83,4 +84,12 @@ export const factoryContracts = {
     communityStake: '0xc49eecf7af055c4dfa3e918662d9bbac45544bd6',
     chainId: 974399131,
   },
+  [ValidChains.Anvil]: {
+    factory: '', //TODO: Fix, this is incorrect
+    communityStake: 'temp', //TODO: Fix, this is incorrect
```

Comment:
Can we update these now or...

Reply:
We can update them now but I am lazy :)

In all seriousness if I update them I will need to check they point to the correct address becuase sometimes our deployment script on the crypto side names them wrong (Like was the case for the lpBondingCurve). Maybe its best to leave it empty and if someone uses it they can go through the hassle of obtaining the addresses and verifying they are correct.